tests/models/test_fairchem.py

```diff
@@ -41,28 +42,20 @@ def ocp_calculator(model_path: str) -> OCPCalculator:
     return OCPCalculator(checkpoint_path=model_path, cpu=False, seed=0)
 
 
-test_fairchem_ocp_consistency = make_model_calculator_consistency_test(
+test_fairchem_ocp_consistency_pbc = make_model_calculator_consistency_test(
     test_name="fairchem_ocp",
-    model_fixture_name="fairchem_model",
+    model_fixture_name="fairchem_model_pbc",
     calculator_fixture_name="ocp_calculator",
-    sim_state_names=consistency_test_simstate_fixtures,
+    sim_state_names=consistency_test_simstate_fixtures[:-1],
     rtol=5e-4,  # NOTE: fairchem doesn't pass at the 1e-5 level used for other models
     atol=5e-4,
 )
 
+# TODO: add test for non-PBC model
```

Comment:
Not clear that the OCP calculator handles non-pbc atoms so unclear how to do the comparison.